packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -98,6 +99,10 @@ export class StripeWebhookHandler extends StripeHandler {
         case 'invoice.payment_failed':
           await this.handleInvoicePaymentFailedEvent(request, event);
           break;
+        case 'product.updated':
+        case 'plan.updated':
+          await this.handleProductUpdate(request, event);
```

Comment:
The same function is used to handle both a product.updated and plan.updated here but the shape of the event objects won't be the same.